mongorestore/mongorestore.go

```diff
@@ -396,6 +399,14 @@ func (restore *MongoRestore) Restore() Result {
 			}
 			return Result{Err: fmt.Errorf("mongorestore target '%v' invalid: %v", restore.TargetDirectory, err)}
 		}
+		preludeFileExists, err := restore.ReadPreludeMetadata(target)
+		if !preludeFileExists {
+			// don't error out here because older dump versions will not prelude.json
```

Comment:
Can you be more specific than “older”? What specific version added prelude.json?